gtsfm/utils/features.py

```diff
@@ -71,39 +71,40 @@ def convert_to_homogenous_coordinates(non_homogenous_coordinates: np.ndarray,) -
     return np.hstack((non_homogenous_coordinates, np.ones((non_homogenous_coordinates.shape[0], 1)),))
 
 
-def convert_to_epipolar_lines(normalized_coordinates_i1: np.ndarray, i2Ei1: EssentialMatrix) -> Optional[np.ndarray]:
+def convert_to_epipolar_lines(coordinates_i1: np.ndarray, i2Fi1: np.ndarray) -> Optional[np.ndarray]:
     """Convert coordinates to epipolar lines in image i2.
 
-    The epipolar line in image i2 is given by i2Ei1 @ x_i1. A point x_i2 is on this line if x_i2^T @ i2Ei1 @ x_i1 = 0.
+    The epipolar line in image i2 is given by i2Fi1 @ x_i1. A point x_i2 is on this line if x_i2^T @ i2Fi1 @ x_i1 = 0.
 
     Args:
-        normalized_coordinates_i1: normalized coordinates in i1, of shape Nx2.
+        coordinates_i1: coordinates in i1, of shape Nx2.
```

Comment:
@ayushbaid i noticed the docstring says `i2Ei1` instead of `i2Fi1`

Reply:
thanks. fixed